src/main/java/mate/academy/webshop/controllers/InjectDataController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.webshop.controllers;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.academy.webshop.lib.Injector;
+import mate.academy.webshop.model.User;
+import mate.academy.webshop.service.UserService;
+
+public class InjectDataController extends HttpServlet {
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy.webshop");
+    private UserService userService = (UserService) INJECTOR.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        userService.create(new User());
+        userService.create(new User());
```

Comment:
you creating Users without any fields and without shoppingCart for them

Reply:
My bad luck. Already optimized.